src/core/handle.ml

```diff
@@ -122,8 +110,30 @@ let handle_cmd : sig_state -> p_command -> sig_state * proof_data option =
         end;
       (* Actually add the symbol to the signature and the state. *)
       out 3 "(symb) %s : %a\n" x.elt pp_term a;
-      (Sig_state.add_symbol ss e p x a impl None, None)
-  | P_rules(rs)                ->
+      Sig_state.add_symbol ss e p x a impl None
```

Comment:
It would be good to have handle_symbol, and thus Sig_state.add_symbol, return the created symbol as well.

Reply:
done